data/sheragi/archaeology missions.txt

```diff
@@ -1057,24 +1062,36 @@ mission "Sheragi Archaeology: The Box 7b"
-			`	As you continue to wander further down the aisle, you find rooms dedicated to the Korath raids, their civil war, and the first contact with them, along with a very poetic recounting of the origins of the Unfettered. Going deeper, you find even more sections: A long one detailing Hai culture on several different worlds and how they shifted with time, another on the fall of the trans-wormhole government and the decision to pull back into a small territory, one on the completion of the Quarg ringworld, one for Hai trade and military culture during their imperial phase, and a very grim-looking room on the Hai Civil Wars. Finally, after a very long walk, you arrive at a room nearly at the end of the aisle, recounting the history of the Sheragi.`
+			`	The place has a very unusual architecture vaguely resembling Earth's Ancient Greece, but clearly not based on any human design. Touches of metal, concrete, glass, and plastic add a modern aspect to its design. Exhibit rooms branch off a long corridor that makes up the majority of the building. The rooms are arranged such that history proceeds backwards as you walk through the museum, with recent events at the front and the earliest at the back.`
+			branch alphas
+				has "Wanderers: Alpha Surveillance A: done"
+			
+			`	One room you enter is dominated by a mural depicting the Hai's history of contact with humanity. What jumps out to you is that the first humans in the mural are shown with green skin. The only green skinned humans you know of are the Alphas. The mural seems to show the Alphas stealing something from the Hai, followed by a series of battles, at the end of which the Alphas are shown retreating into the wormhole. Ships then come back through the wormhole, only this time carrying normal humans. The Hai in the mural appear apprehensive at first, but it is clear that a friendship was quickly established.`
+			`	It is surprising and unfortunate to learn that the first contact the Hai ever had with humanity was with the Alphas.`
+				goto continue
+			
+			label alphas
+			`	One room you enter is dominated by a mural depicting the Hai's history of contact with humanity. It begins with the Alphas, just as Ambassador Hiyamaharu told you about. The mural seems to show the Alphas stealing something from the Hai, followed by a series of battles, at the end of which the Alphas are shown retreating into the wormhole. Ships then come back through the wormhole, only this time carrying normal humans. The Hai in the mural appear apprehensive at first, but it is clear that a friendship was quickly established.`
```

Comment:
Previously, this conversation mentioned the Alphas as if the player already knew that the Hai had contact with them, but that may not be the case. The player doesn't learn that the Alphas came to Hai space until during the Wanderer campaign, while this mission can be done prior to the Wanderer campaign. This branch changes the language slightly based on what the player already knows.